pkg/cache/sql/db/client.go

```diff
@@ -22,6 +23,8 @@ import (
 const (
 	// InformerObjectCacheDBPath is where SQLite's object database file will be stored relative to process running lasso
 	InformerObjectCacheDBPath = "informer_object_cache.db"
+
+	informerObjectCachePerms fs.FileMode = 0600
```

Comment:
nit, we could make it explicit that this is a literal octal:

```suggestion
	informerObjectCachePerms fs.FileMode = 0o600
```